src/renderer/ducks/beacon/sagas.ts

```diff
@@ -94,9 +102,31 @@ function* startLocalBeaconSaga({
     cgLogger.info("Starting local docker beacon node & http://localhost:", rpcPort);
     const eth1QueryLimit = 200;
     if (pullSuccess) {
-        const cors = process.env.NODE_ENV !== "production" ? " --http-allow-origin http://localhost:2003 " : " ";
-        switch (network) {
-            default:
+        switch (client) {
+            case "teku": {
+                const cors =
```

Comment:
we can assume we are always running local node. So setting cord to "*" is fine for all clients

Reply:
CORS is only required for testing development